File: A4.Empower-Git/A4.Empower/A4.BAL/Performance/HrViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace A4.BAL
{
    public class HrViewModel
    {
        public bool IsPerformanceStart { get; set; }
        public bool IsConfigurationSet { get; set; }
        public bool IsMidYearEnabled { get; set; }
        public bool IsMidYearReviewCompleted { get; set; }
        public List<HrViewList> lstManager { get; set; }
        public List<HrViewList> lstEmployee { get; set; }
        public int totalCount { get; set; }
    }

    public class HrViewList
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Designation { get; set; }
        public string Group { get; set; }
        public string Status { get; set; }       
    }
}

File: ClientApp/src/app/models/performance/hr-view/hr-view.model.ts
export class HrView {
    constructor() {
        this.lstManager = new Array<hrViewList>();
        this.lstEmployee = new Array<hrViewList>();

    }
    public isPerformanceStart: boolean;
    public isConfigurationSet: boolean;
    public isMidYearEnabled: boolean; 
    public isMidYearReviewCompleted: boolean;
    public lstManager: hrViewList[];
    public lstEmployee: hrViewList[];
    public totalCount: number;
}


export class hrViewList {

    public id: string;
    public name: string;
    public designation: string;
    public group: string;
    public status: string;
}


